cinema/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```

Comment:
Consider registering your models here using `admin.site.register(YourModel)` to make them accessible in the Django admin interface.